pkg/lib/cockpit-components-plot.jsx

```diff
@@ -258,7 +258,7 @@ export const ZoomControls = ({ plot_state }) => {
         <div>
             <Dropdown
                 isOpen={isOpen}
-                toggle={<DropdownToggle onToggle={setIsOpen}>{format_range(zoom_state.x_range)}</DropdownToggle>}
+                toggle={<DropdownToggle onToggle={(_, isOpen) => setIsOpen(isOpen)}>{format_range(zoom_state.x_range)}</DropdownToggle>}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18898-20230607-143755-90f0272e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)